packages/firestore/src/core/database_info.ts

```diff
@@ -29,13 +29,16 @@ export class DatabaseInfo {
    * @param ssl Whether to use SSL when connecting.
    * @param forceLongPolling Whether to use the forceLongPolling option
    * when using WebChannel as the network transport.
+   * @param autoDetectLongPolling Whether to use the autoDetectLongPolling
```

Comment:
```suggestion
   * @param autoDetectLongPolling Whether to use the detectBufferingProxy
```

This matches the name of the WebChannel option.